helios-testing/src/main/java/com/spotify/helios/testing/HeliosDeploymentResource.java

```diff
@@ -64,8 +65,8 @@ public void before() throws Throwable {
     Polling.awaitUnchecked(30, TimeUnit.SECONDS, new Callable<Boolean>() {
       @Override
       public Boolean call() throws Exception {
-        final HostAndPort hap = deployment.address();
-        final SocketAddress address = new InetSocketAddress(hap.getHostText(), hap.getPort());
+        final URI uri = Iterables.getOnlyElement(deployment.uris());
```

Comment:
if the Set in `uris()` had more than one elemnent, `Iterables.getOnlyElement()` will throw an exception. Is it really necessary to fail if there is > 1 URI?